### PR TITLE
 Create header and scrollble filter 

### PR DESCRIPTION
**Pull request by MostafaBalousha123**
1-Create header and scrollble filter (horizontal row of buttonsl) for community page
2-Add image file for the page
3-add style
relate #8

### CODE REVIEW

**Review by Mo3en-ps — DISMISSED** [review]

File: community/community.html [hunk 1]
```diff
@@ -4,8 +4,41 @@
...
-  <body></body>
+  <body>
+    <!--Start Header-->
+    <header class="header">
+      <p class="Continer-for-link-a"><a href="#"> &larr; </a></p>
+      <p class="header-community">Community</p>
+    </header>
+    <!--End Header-->
+    <main>
+      <section class="section-for-filter">
```
Comment (techhana2014) [inline community/community.html:24]: I think you can put this section in header

File: community/community.html [hunk 2]
```diff
@@ -4,8 +4,41 @@
...
+        <div id="my-Filter-Container">
```
Comment (techhana2014) [inline community/community.html:25]: Use nav instead of div for your navigation bar

File: community/community.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  }
+  
+  header p.Continer-for-link-a {
+    display: inline-block;
+    font-size: 20px;
+    text-align: end;
+    width: 10%;
+  }
+  /*** End header***/
+  #my-Filter-Container {
```
Comment (techhana2014) [inline community/community.css:40]: I think that is better to make class name as camelCase for all css

**Review by techhana2014 — CHANGES_REQUESTED** [review]

File: community/community.html [hunk 4]
```diff
@@ -4,8 +4,43 @@
...
+    <title>Food App</title>
+    <link rel="stylesheet" href="community.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lobster&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body></body>
```
Comment (Mo3en-ps) [inline community/community.html:10]: Is this extra body attribute?!!

**Review by Mo3en-ps — CHANGES_REQUESTED** [review]

**Review by techhana2014 — APPROVED** [review]

File: community/community.html (hunk 4 above)
Comment (techhana2014) [reply community/community.html:10]: No There is no problem, it only appeared when rearranging the codes

**Review by MohamedSbaih — APPROVED** [review]

**Review by Mo3en-ps — APPROVED** [review]

**Pull request merged**